app/assets/stylesheets/application.tailwind.css

```diff
@@ -0,0 +1,13 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/*
+
+@layer components {
+  .btn-primary {
+    @apply py-2 px-4 bg-blue-200;
+  }
+}
+
+*/
```

Comment:
>Commented code is there which is not required.

@aliraza4638 we shouldn't have any unused commented code as we discussed already ?